src/test/integrationSuiteSetup.ts

```diff
@@ -36,4 +37,5 @@ beforeEach(async () => {
 afterEach(async () => {
   await cleanupDatabaseForCurrentWorker();
   jest.restoreAllMocks();
+  nock.cleanAll();
```

Comment:
This line seems to result in some wild errors int ests but *ONLY* after c08557b9caac64768119ba8700ca68c47a20d4de

That's the commit that removed auth from a the base fields... I haven't figured out what exactly is going on.  I believe the jwt library is using nock to override jwt interactions, and so it makes sense that `cleanAll` might conflict with it.  It does not make nearly as much sense that the conflict would only manifest after removing auth from a single endpoint, however.

I'll keep digging.

Reply:
OK -- looks like moving the `mockJwks` to the beforeEach / afterEach instead of beforeAll / afterAll does the trick for the local issues.

Reply:
That makes sense! Good find.